docs/census-open-data-registry.yaml

```diff
@@ -0,0 +1,37 @@
+Name: CZ CELLxGENE Discover Census
+Description: The Census provides efficient computational tooling to access, query, and analyze all single-cell RNA data from CZ CELLxGENE Discover.
+Documentation: https://chanzuckerberg.github.io/cellxgene-census/
+Contact: cellxgene@chanzuckerberg.com
+ManagedBy: "[Chan Zuckerberg Initiative](http://www.chanzuckerberg.com/)"
+UpdateFrequency: New releases are published weekly. LTS releases are published every 6 months.
+Tags:
+  - single-cell
+  - genomics
+  - computational biology
+License: MIT License
+Resources:
+  - Description: CELLxGENE Census Data
```

Comment:
CELLxGENE Discover Census 

Need to ensure we comply with branding

Reply:
CZ CELLxGENE Discover Census?

I'll check with Tiffany...

Reply:
you are correct - apologies.

See our main web site for the approved naming.  https://chanzuckerberg.github.io/cellxgene-census/

Reply:
Updated.